common/src/main/java/com/netflix/conductor/common/utils/ConstraintParamUtil.java

```diff
@@ -56,12 +56,6 @@
     private static List<String> extractParamPathComponentsFromString(String key, String value, String taskName, WorkflowDef workflow) {
         ArrayList<String> errorList = new ArrayList<>();
 
-        if (StringUtils.isEmpty(value)) {
```

Comment:
`StringUtils.isEmpty` checks for both null and empty values. Removing this check altogether removes null check validation as well. Please update so that only empty strings are permitted but null values are validated against.

Reply:
done :)